doc/SANDBOXED_BUILDS.md

```diff
@@ -6,7 +6,7 @@ environments.
 ## Docker
 
 A simple way to perform a from-source build is to use the
-[openassetio-build docker image.](https://github.com/OpenAssetIO/OpenAssetIO/pkgs/container/openassetio-build)
+[openassetio-build docker image.](https://github.com/OpenAssetIO/OpenAssetIO/pkgs/container/openassetio-build/)
```

Comment:
[question] IIRC you said this wasn't necessary?

Reply:
Ah yeah it's not, forgot to remove, will do.
It didn't occur to me that his url in particular could be flaky, but I guess if you saw it too then yeah it might be.